### PR TITLE
Assigment Uploded

### PR DESCRIPTION
**Pull request by Amit727619**

### CODE REVIEW

**Review by pritikekan — APPROVED** [review]

**Review by pooja-nagare — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Review by jyoti-ing — APPROVED** [review]

File: amit/html/Assigment1/ASSIn1.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <h3>
+           <b>Introduction</b> 
+        </h3>
+        <p> HTML is a standard markup language for Web Pages.</p>
+        <h3> 
+        <b>HTML Editor </b>
+        </h3>
+        <h4><b>Notepad or TextEdit.</b></h4>
+        <p> 
+        Web pages can be created and modified by using professional HTML editors.
```
Comment (KomalShinde813) [inline amit/html/Assigment1/ASSIn1.html]: check indentation this line should be under <p> tag

File: amit/html/Assigment1/ASSIn1.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+However, for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).
+        </p>
+
+     </body>
+</html>
```
Comment (KomalShinde813) [inline amit/html/Assigment1/ASSIn1.html]: extra line required at the end of file

File: amit/html/Assigment2/ASSIn1.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+
+<html>
+    <head>
+    <title>this is my first html</title>
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: check indentation

Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: check indentation

File: amit/html/Assigment2/ASSIn1.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    </head>
+           <h1>
+           <center>Google</center>
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: check indentation

File: amit/html/Assigment2/ASSIn1.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+           </h1>
+    This article is about the company.
+    <h1>
+    From the garbage to the Googleplex
+    </h1>
+    <p>
+     The Google story begins in 1995 at <b>Stanford university.</b><br>
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: make indentation changes in each and every file

File: amit/html/Assigment2/ASSIn1.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    </p>
+    They called this search engine <font color="black" style=background-color:blue>.Backrub</font>
+    <hr>
+    </hr>
+    <h2>Initial public offering </h2><br>
+    Google's Initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+    At that time Larry Page,Sergey Brin,and Eric Schmidt agreed to work together at Google for30 20years,Until the year 2024.<br>
+    <center><font color="green" ><b> Thank You!!</b></font></center>
+    </html>
+
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: required only one extra line at the end of file check in every file

File: amit/html/Assigment3/Amit3_2.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  </body>
+ </html>
+
+
+ 
+
+
+
+
+
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: remove extra lines

File: amit/html/Assigment3/Amit3_3.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  <style>
+  body  {background-color: powderblue;}
+  h1    {color: blue;}
+  p     {color: red;}
+
+   </style>
+<div style="text-align: center;">
+    <h1>AmberSoft</h1><br>
+
+
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_3.html]: remove extra lines

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by anuja1603 — APPROVED** [review]

File: amit/html/Assigment2/ASSIn1.html [hunk 9]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+ <html>
+    <head>
+          <title>this is my first html</title>
+    </head>
+           <h1> <center> Google </center> </h1>
+        This article is about the company.
+           <h1> From the garbage to the Googleplex </h1>
+       <p> The Google story begins in 1995 at <b>Stanford university.</b><br> <b><i>Larry Page</i></b> was considering stanford for grad School and <b><i>Sergey Brin,</i></b> a student there,was assigned to Show him around.
+       </p>
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: indentation is not proper

File: amit/html/Assigment2/ASSIn1.html [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+       <p> By Some accounts,they disagreed about nearly everything during that first meeting.but by the following year they struck a <mark> partnership.</mark><br>
+           Working from their dorm rooms,they built a search engine that used links to determine the importance of individual pages on the <font color="blue">World Wide Web.</font>
+       </p>
+           They called this search engine <b>Backrub</b>
+    <hr>
+    </hr>
+           <body style="background-color: pink;"></body>
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: check indentation in this file

File: amit/html/Assigment2/ASSIn1.html [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+            Working from their dorm rooms,they built a search engine that used links to determine the importance of individual pages on the <font color="blue">World Wide Web.</font>
+        </p>They called this search engine <font color="black" style=background-color:blue>.Backrub</font>
+    <hr>
+    </hr>
+          <h2>Initial public offering </h2><br>
+         Google's Initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+         At that time Larry Page,Sergey Brin,and Eric Schmidt agreed to work together at Google for30 20years,Until the year 2024.<br>
+           <center><font color="green" ><b> Thank You!!</b></font></center>
+    </html>
+
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: remove extra lines which is present at the end of file only one line space expected after </html> tag

File: amit/html/Assigment3/Amit3_1.html [hunk 12]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang ="en">
+<head>
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: check indentation head is a child tag of html so it should be inside html

File: amit/html/Assigment3/Amit3_1.html [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+    <meta charset="UTF-8">
+    <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+    <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
+</head>
+    <body>
+              <h1> AmberSoft</h1>
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: check extra space here need 4 spaces from parent tag

File: amit/html/Assigment3/Amit3_1.html [hunk 14]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang ="en">
+<head>
+    <meta charset="UTF-8">
+    <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+    <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: space should be after comma so remove space which is present before the comma

Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: check extra space before the content

File: amit/html/Assigment3/Amit3_2.html [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+        <a href="http://www.MI-MEAN"> MI-MEAN</a>
+        <a href="http://www.contact"> contact</a>
+     </div>
+     </div>
+         <h2>Home Page</h2>
+     </div>
+
+    <div  style="display: flex;">
+    <div style="width: 33.33%;">
+              <img src="brige.jpeg" alt="image" height="450px" width="300px" style="margin: 10px;">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: check indentation properly

File: amit/html/Assigment4/Amit4_3.html [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+            <tr>
+                <td>Wednesday</td>
+                <td class="b">8:00 a.m.</td>
+                <td class="d">12:00 p.m.</td>
+                <td>XSL Formatting Object</td>
+            </tr>
+       </body>
+    </table>
+</body>
+</html>
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_3.html]: add extra line space at the end of the file

File: amit/html/Assigment4/Amit4_3.html [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+        .c{
+            background-color: rgb(140, 140, 236);
+        }
+        .d{
+            background-color: cyan;
+        }
+
+    </style>
+
+<body>
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_3.html]: check indentation

File: amit/html/Assigment4/Amit4_3.html [hunk 18]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_3.html]: correct the indentation in each and every file first

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: amit/html/Assigment2/ASSIn1.html [hunk 19]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+  <html>
+     <head>
+        <title>this is my first html</title>
+     </head>
+        <h1><center> Google </center> </h1> This article is about the company.
+        <h1>From the garbage to the Googleplex </h1>
+           <p>The Google story begins in 1995 at
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: check indentation

File: amit/html/Assigment3/Amit3_1.html [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+ <html lang ="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+      <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
+   </head>
+       <body>
+          <h1> AmberSoft</h1>
+           <div>
+            <ol style="list-style-type: number">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: check indentation this should under <div>

File: amit/html/Assigment3/Amit3_1.html [hunk 21]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+ <html lang ="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+      <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
+   </head>
+       <body>
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: check indentation this line should be exact below </head>

File: amit/html/Assigment3/Amit3_2.html [hunk 22]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+ <html lang ="en">
+   <head>
+     <meta charset="UTF-8">
+     <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+     <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
+   </head>
+    <title> images</title>
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: <title should be inside head

File: amit/html/Assigment3/Amit3_2.html [hunk 23]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+ <html lang ="en">
+   <head>
+     <meta charset="UTF-8">
+     <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+     <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: remove extra space present before content and remove space before , give it after ,

File: amit/html/Assigment3/Amit3_2.html [hunk 24]
```diff
@@ -0,0 +1,40 @@
...
+        </div>
+        <div style="text-align: center">
+          <a href="http://www.home.com" > Home </a>
+          <a href="http://www.MI-MEAN"> MI-MEAN</a>
+          <a href="http://www.contact"> contact</a>
+        </div>
+             <h2>Home Page</h2>
+        </div>
+         <div  style="display: flex;">
+          <div style="width: 33.33%;">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: check indentation properly

File: amit/html/Assigment3/Amit3_2.html [hunk 25]
```diff
@@ -0,0 +1,40 @@
...
+                 Home Page.Amber Academy is an educational program that aims to share its educational resources with the entire gaming industry: from students, to other peeps, and last but not last, with our colleagues.This is the Home Page.Amber Academy is an educational program that aims to share its educational resources with the entire gaming industry: from students, to other peeps, and last but not last, with our colleagues.
+                 his is the Home Page.Amber Academy is an educat
+              </p>
+         <div style="width: 33.33%;">
+            <img src="brige3.jpeg" alt="image" height="450px" width="300px" style="margin: 10px;">
+         </div>
+        </div>
+        <div>
+            <h2 style="text-align: left;">Home page subheading</h2>
+             <p> This is home page subheading. this  is home page subheading. this is   is home page subheading. this is
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: check indentation properly

Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: check extra spaces present in this line

File: amit/html/Assigment3/Amit3_3.html [hunk 26]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang ="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+    <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
+  </head>
+   <title> images</title>
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_3.html]: title should be inside head make this change in every file

File: amit/html/Assigment3/Amit3_3.html [hunk 27]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang ="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta hhtp-equiv= "X-UA-Compatible" content="IE=edge">
+    <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_3.html]: remove extra space present before content and remove space before , make this change in all files

File: amit/html/Assigment3/Amit3_3.html [hunk 28]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang ="en">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_3.html]: remove space before = make check all files for this change

File: amit/html/Assigment3/Amit3_3.html [hunk 29]
```diff
@@ -0,0 +1,48 @@
...
+    <meta name="viewport"  content= "width=device-width ,initial-scale=1.0">
+  </head>
+   <title> images</title>
+    <style>
+      body  {background-color: powderblue;}
+      h1    {color: blue;}
+      p     {color: red;}
+    </style>
+   <div style="text-align: center;">
+         <h1>AmberSoft</h1><br>
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_3.html]: check indentation in each and every file

File: amit/html/Assigment3/Amit3_3.html [hunk 30]
```diff
@@ -0,0 +1,48 @@
...
+          <li> 
+              Pawan 
+          </li> 
+          <li> 
+             Anitket
+          </li> 
+     </ol>   
+    </div>
+   <div>
+     <ol style="list-style-type: lower-roman">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_3.html]: check indentation and check all files for indentation change

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: amit/html/Assigment1/ASSIn1.html [hunk 31]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Assignment 1 </title> 
+    </head>
+      <body>
+         <h1> HTML </h1>
+        <h3><b>Introduction</b></h3>
+            <p>HTML is a standard markup language for Web Pages.</p>
```
Comment (KomalShinde813) [inline amit/html/Assigment1/ASSIn1.html]: this line should be below h3 tag

File: amit/html/Assigment1/ASSIn1.html [hunk 32]
```diff
@@ -0,0 +1,16 @@
...
+        <h3> <b>HTML Editor </b></h3>
+        <h4><b>Notepad or TextEdit.</b></h4>
+            <p>Web pages can be created and modified by using professional HTML editors.However, for learning HTML we recommend a simple
```
Comment (KomalShinde813) [inline amit/html/Assigment1/ASSIn1.html]: this tag should be exact below h4

File: amit/html/Assigment2/ASSIn1.html [hunk 33]
```diff
@@ -0,0 +1,61 @@
...
+         </hr>
+         <h3>
+            Initial public offering
+         </h3>
+         <p>
+            Google's initial public offering (IPO) took place five years later, on 19<sup>th</sup> August 2004. <br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <u>20</u>
+            years, until the year 2024.
+         </p> 
+
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: remove extra line

File: amit/html/Assigment2/ASSIn1.html [hunk 34]
```diff
@@ -0,0 +1,61 @@
...
+         <h3>
+            Initial public offering
+         </h3>
+         <p>
+            Google's initial public offering (IPO) took place five years later, on 19<sup>th</sup> August 2004. <br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <u>20</u>
+            years, until the year 2024.
+         </p> 
+       
+            <h1> <center> Google <center> </h1>
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: this tag should exact below p tag

File: amit/html/Assigment2/ASSIn1.html [hunk 35]
```diff
@@ -0,0 +1,61 @@
...
+            Initial public offering
+         </h3>
+         <p>
+            Google's initial public offering (IPO) took place five years later, on 19<sup>th</sup> August 2004. <br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <u>20</u>
+            years, until the year 2024.
+         </p> 
+       
+            <h1> <center> Google <center> </h1> 
+               <p>
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: check indentation

File: amit/html/Assigment3/Amit3_1.html [hunk 36]
```diff
@@ -0,0 +1,55 @@
...
+      <meta name="viewport" content="width=device-width ,initial-scale=1.0">
+      <title>Lists</title>
+    </head>
+    <body>
+        <div>
+          <h1>AmberSoft</h1>
+            <ol style="list-style-type: number">MI-MEAN
+               <ol style="list-style-type: lower-roman">
+                 <li>MI-MEAN-6</li>
+                 <ul style= "list-style-type:disc">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: space required after : in style

File: amit/html/Assigment3/Amit3_1.html [hunk 37]
```diff
@@ -0,0 +1,55 @@
...
+                    <li> Shweta </li> 
+                    <li> Ajay </li>
+                    <li> Trupti </li>
+                 </ul>
+                 <li>MI-MEAN-7</li>                
+                 <ul style= "list-style-type:disc">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: space required after : in style

File: amit/html/Assigment3/Amit3_1.html [hunk 38]
```diff
@@ -0,0 +1,55 @@
...
+                    <li> Sagar </li> 
+                    <li> Umesh </li>
+                 </ul>
+                 <li>MI-MEAN-8</li>        
+                 <ul style= "list-style-type:disc">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: space required after : in style

File: amit/html/Assigment3/Amit3_1.html [hunk 39]
```diff
@@ -0,0 +1,55 @@
...
+                     <li> Mihika </li> 
+                     <li> Pallavi </li>
+                     <li> Reva </li> 
+                 </ul>
+                 <li>MI-MEAN-9</li>   
+                 <ul style= "list-style-type:disc">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: space required after : in style

File: amit/html/Assigment3/Amit3_1.html [hunk 40]
```diff
@@ -0,0 +1,55 @@
...
+                     <li> Anil </li> 
+                     <li> Manoj</li>         
+                  </ul>   
+                </ol>
+                <li> System </li>
+                  <ol style="list-style-type: lower-roman">
+                     <li> System 1 </li>
+                      <ul style= "list-style-type:disc">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: space required after : in style

File: amit/html/Assigment3/Amit3_1.html [hunk 41]
```diff
@@ -0,0 +1,55 @@
...
+                        <li> Anu </li> 
+                        <li> Dipti </li>   
+                      </ul>  
+                     <li> System 2 </li>    
+                     <ul style= "list-style-type:disc">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: space required after : in style

File: amit/html/Assigment3/Amit3_2.html [hunk 42]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+ <html lang="en">
+   <head>
+     <meta charset="UTF-8">
+     <meta hhtp-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width ,initial-scale=1.0">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: remove space before , add space after ,

File: amit/html/Assigment3/Amit3_2.html [hunk 43]
```diff
@@ -0,0 +1,33 @@
...
+     <title>document</title>
+   </head>
+   <body>
+      <div style="text-align: center;">
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_2.html]: where this div tag closed?

File: amit/html/Assigment3/Amit3_3.html [hunk 44]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta hhtp-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> images</title>
+  </head>
+   <style>
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_3.html]: this should exact below head tag

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: amit/html/Assigment1/ASSIn1.html [hunk 45]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Assignment 1</title> 
+    </head>
+      <body>
+         <h1>HTML</h1>
```
Comment (KomalShinde813) [inline amit/html/Assigment1/ASSIn1.html]: check indentation

File: amit/html/Assigment2/ASSIn1.html [hunk 46]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+  <html>
+     <head>
+        <title>this is my first html</title>
+     </head>
+         <h1><center> Google </center> </h1>
```
Comment (KomalShinde813) [inline amit/html/Assigment2/ASSIn1.html]: check indentation from this line to before </html>

File: amit/html/Assigment3/Amit3_1.html [hunk 47]
```diff
@@ -0,0 +1,55 @@
...
+                        <li> Triveni </li> 
+                        <li>Kalyani </li>  
+                     </ul> 
+                   </ol>
+                </li>
+            </ol>      
+        </div> 
+     </body>
+ </html>
+
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_1.html]: remove extra lines only one line space expected

File: amit/html/Assigment3/Amit3_4.html [hunk 48]
```diff
@@ -0,0 +1,57 @@
...
+                color: navy;
+                background-color: transparent;
+                font-style: normal;
+            }
+
+            a:hover {
+                color: palevioletred;
+                background-color: transparent;
+                font-style: italic;
+            }
```
Comment (KomalShinde813) [inline amit/html/Assigment3/Amit3_4.html:22]: add one line space here or remove spaces above make sure its consistent in all files

File: amit/html/Assigment4/Amit4_2.html [hunk 49]
```diff
@@ -0,0 +1,127 @@
...
+                    <th>Wednesday</th>
+                    <th>Thursday</th>
+                    <th>Friday</th>
+                </tr>
+                <tr>
+                    <th>1:00</th>
+                    <td rowspan="2">Office</td>
+                    <td rowspan="1"></td>
+                    <td rowspan="2">Office</td>
+                    <td rowspan="8" ; class="adc">
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_2.html]: remove space before ;

File: amit/html/Assigment4/Amit4_2.html [hunk 50]
```diff
@@ -0,0 +1,127 @@
...
+                    <td rowspan="3">
+                        EC <br />
+                        Meeting
+                    </td>
+                    <td>&nbsp;</td>
+                    <td>&nbsp;</td>
+                </tr>
+                <tr>
+                    <th>3:30</th>
+                    <td rowspan="3" ; class="adc">
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_2.html]: remove space before ;

File: amit/html/Assigment4/Amit4_2.html [hunk 51]
```diff
@@ -0,0 +1,127 @@
...
+                        CSE <br />
+                        205;
+                    </td>
+                    <td rowspan="3" ; class="adc">
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_2.html]: remove space before ;

File: amit/html/Assigment4/Amit4_2.html [hunk 52]
```diff
@@ -0,0 +1,127 @@
...
+                        CSE <br />
+                        205
+                    </td>
+                    <td rowspan="3" ; class="adc">
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_2.html]: remove space before ;

File: amit/html/Assigment4/Amit4_3.html [hunk 53]
```diff
@@ -0,0 +1,82 @@
...
+    </style>
+    <body>
+        <table style="width: 50%" border="2" cellpadding="23">
+            <tr>
+                <th rowspan="3">DAY</th>
+                <th colspan="5">seminar</th>
+            </tr>
+            <tr>
+                <th colspan="2">schedule</th>
+                <th colspan="3" ; rowspan="2">Topic</th>
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_3.html]: remove space before ;

File: amit/html/Assigment4/Amit4_3.html [hunk 54]
```diff
@@ -0,0 +1,82 @@
...
+            </tr>
+            <tr>
+                <th>Begin</th>
+                <th>End</th>
+            </tr>
+            <tr>
+                <td rowspan="2">Monday</td>
+                <td rowspan="2" ; class="b">8:00 a.m.</td>
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_3.html]: remove space before ;

File: amit/html/Assigment4/Amit4_3.html [hunk 55]
```diff
@@ -0,0 +1,82 @@
...
+                <td rowspan="2" ; class="c">5:00 p.m.</td>
```
Comment (KomalShinde813) [inline amit/html/Assigment4/Amit4_3.html]: remove space before ;

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

File: amit/html/Assigment4/Amit4_2.html (hunk 51 above)
Comment (Amit727619) [reply amit/html/Assigment4/Amit4_2.html]: mam when i apply prettier then this automaticaly happen

File: amit/html/Assigment4/Amit4_3.html (hunk 55 above)
Comment (Amit727619) [reply amit/html/Assigment4/Amit4_3.html]: this is also after prettier happen i think its prettier format

File: amit/html/Assigment3/Amit3_1.html (hunk 47 above)
Comment (Amit727619) [reply amit/html/Assigment3/Amit3_1.html]: sir told 1 line is nessory after complte code

**Review by Pawan1005 — APPROVED** [review]

**Review by KomalShinde813 — APPROVED** [review]

**Pull request merged**